conf/upf.json

```diff
@@ -64,7 +66,9 @@
         "nb_dst_ip": "172.17.0.1",
         "" : "nb_dst_ip: CPHostname",
         "hostname": "spgwc",
-        "prom_port": "8080"
+        "prom_port": "8080",
+        "" : "dnn:internet",
```

Comment:
same comment.

Reply:
commented for now. will configure when needed. tested with configuration enabled in case of 5gc